src/analog/commands/uplink/updateRun.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Command to run the update on the device.
+ * This command is part of update procedure.
+ *
+ * @packageDocumentation
+ *
+ * @example
+ * ```js
+ * import * as updateRun from 'jooby-codec/analog/commands/uplink/updateRun.js';
+ *
+ * // empty response
+ * const bytes = [];
+ * const parameters = updateRun.fromBytes(bytes);
+ *
+ * // this command doesn't have any parameters
+ * console.log(parameters);
+ * // output:
+ * undefined
```

Comment:
```suggestion
 * {}
```

?

Reply:
based on result from fromBytes in this file

Reply:
fixed